micro-infra-spring-base/src/main/java/com/ofg/infrastructure/web/logging/HttpDataExtractor.java

```diff
@@ -126,19 +127,15 @@ static String extractContent(ClientHttpResponse response){
         ByteArrayOutputStream output = new ByteArrayOutputStream();
         try{
             StreamUtils.copy(response.getBody(), output);
-        } catch (Exception ex) {
-            throw new IllegalStateException("Error extractContent", ex);
-        }
+        } catch (Exception ex) {}
```

Comment:
Don't do things like this. Don't we have exactly error which we throw when stream is with no body ?